src/ui/SmartSnippet/SmartSnippetCollapsibleSuggestion.ts

```diff
@@ -171,13 +171,21 @@ export class SmartSnippetCollapsibleSuggestion {
     }
   }
 
+  private updateIFrameExpanded() {
+    const iframe = this.snippetAndSourceContainer.find('iframe');
+    if (!iframe) {
+      return;
+    }
+    this.expanded ? iframe.removeAttribute('tabindex') : iframe.setAttribute('tabindex', '-1');
+  }
+
   private updateExpanded() {
     this.checkbox.setAttribute('aria-expanded', this.expanded.toString());
     this.checkbox.setHtml(this.expanded ? SVGIcons.icons.arrowUp : SVGIcons.icons.arrowDown);
-    this.collapsibleContainer.setAttribute('tabindex', `${this.expanded ? 0 : -1}`);
```

Comment:
Being able to tab onto the answer is not necessary for accessibiltiy. What's important is being able to select it with the accessibility focus.